src/xgov-dapp/src/features/vote/VotingInstructions.tsx

```diff
@@ -26,8 +26,10 @@ export const VotingInstructions = ({ voteWeight }: { voteWeight: number }) => {
             proposals.
           </strong>
         </Typography>
+        <Typography className="mb-3"></Typography>
         <Typography>
-          <strong>Once you cast your votes, you cannot change them.</strong>
+          <strong>Once you cast your votes, you cannot change them.</strong> Proposals that have <strong>PASSED</strong> cannot receive
+          further votes. {/*Please choose a different proposal to allocate your votes to.*/}
```

Comment:
why the commented out part?

Reply:
Want to see what the PO thinks for verbiage, the commented out section is the exact wording in the ticket. I took some liberty to moved it to the Dialog